README.md

```diff
@@ -136,6 +138,78 @@ Writing a new benchmark fixture would generally look like:
 
 The `benchmark_time` fixture provides a fairly simple example.
 
+## Investigating performance regressions
+
+It is not always obvious what the cause of a seeming performance regression is.
+It could be due to a new version of a direct or transitive dependency,
+and it could be due to a change in the Coiled platform.
+But often it is due to a change in `dask` or `distributed`.
+If you suspect that is the case, Coiled's `package_sync` feature combines well with the benchmarking infrastructure here and `git bisect`.
+
+The following is an example workflow which could be used to identify a specific commit in `dask` which introduced a performance regression.
+This workflow presumes you have two terminal windows open, one with the `coiled-runtime` test suite,
+and one with a `dask` repository with which to drive bisecting.
+
+#### Create your software environment
+
+You should create a software environment which can run this test suite, but with an editable install of `dask`.
+You can do this in any of a number of ways, but one approach coule be
+```bash
+conda env create -n test-env --file ci/environment.yml  # Create a test environment
+conda activate test-env  # Activate your test environment
+pip install .  # Install the `coiled-runtime` metapackage dependencies.
+(cd <your-dask-dir> && pip install -e .)  # Do an editable install for dask
+```
+
+#### Start bisecting
+
+Let's say the current `HEAD` of `dask` is known to be bad, and `$REF` is known to be good.
```

Comment:
I wonder if adding some info about how to get a list of commits is worth it here. Something like 

```suggestion
Let's say the current `HEAD` of `dask` is known to be bad, and `$REF` is known to be good. If you are looking at an upstream run where you have access to the static page, you can check the dates of reported for each run git log with the corresponding dates to get a list of commits as a reference to use in the bisecting process.

`git log --since='2022-08-15 14:15' --until='2022-08-18 14:15' --pretty=oneline`
```

Reply:
Sounds good!